### PR TITLE
fix the failed Chinese unit tests.

### PR DESCRIPTION
**Pull request by rubio41**

### CODE REVIEW

File: Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs [hunk 1]
```diff
@@ -143,24 +143,9 @@ public void TestAge()
...
                       "10 Month");
 
-            /*
-            BasicTest(model,
-                      "如果我们在十二月份开始，会有三周时间。",
-                      "3 Week");
-
-            BasicTest(model,
-                      "这封邮件已经发送出去15天了，回信在哪里?",
-                      "15 Day");
```
Comment (yiylin) [inline Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs]: Why remove these tests

Comment (tellarin) [inline Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs]: If the sentences in Chinese don't make sense. Please add new ones for similar cases to be tested.

Comment (rubio41) [inline Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs]: changed these test cases. The original ones doesn't make sense in Chinese.

Comment (tellarin) [inline Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs]: Please add new ones for similar cases to be tested. @rubio41 We need tests covering month and day.

Comment (rubio41) [inline Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs]: changed to new test cases.

Comment (tellarin) [inline Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs]: Hi @rubio41, I still don't see tests for day and week there. Please add tests to cover those cases (as in the removed ones). Thanks.

**Comment by tellarin** [comment]
Approved after unit tests added.

**Pull request merged**